### ISSUE TITLE
Ubuntu 24.04 Upgrade

### ISSUE THREAD

**Issue by DawoudSheraz** [overhangio/openedx-scorm-xblock#82]
- Ensure CI uses the latest ubuntu
- Ensure any documentation/references are updated

See https://github.com/overhangio/tutor/issues/1113 for context. Because scorm is installed in platform and does not really work standalone, the main focus of the issue is to ensure that xblock works as expected on Ubuntu 24.04.

**Comment by Danyal-Faheem** [comment]
Update: #85 has been merged into nightly which means that all the CI use the latest ubuntu now.

The testing part will continue when the changes for Ubuntu 24.04 are merged upstream.

**Comment by Danyal-Faheem** [comment]
Update: After #88 is merged, this issue can be closed as scorm-xblock has been verified as working with the latest Ubuntu-24.04 changes made to the openedx Dockerfile in tutor-core.
